recipe/run_test.py

```diff
@@ -1,10 +0,0 @@
-import fsspec
-
-from PIL import Image
-
-
-# Test JPEG2k
-with fsspec.open("https://www.fnordware.com/j2k/relax.jp2") as f:
-  image = Image.open(f)
-  image.load()
```

Comment:
This was already dropped in commit ( https://github.com/conda-forge/pillow-feedstock/pull/132/commits/2e44e1412db935e5302c184af7e244f76f44c9dd )